src/scss/_layout.scss

```diff
@@ -10,6 +10,7 @@
         position: relative;
 
         display: flex;
+        flex-direction: column;
```

Comment:
By default it was direction: row and I needed this to be column to stack my "recommended by" and the CAO logo